plugins/search-elasticsearch-rest/src/main/java/com/thinkbiganalytics/search/indexing/ElasticSearchMetadataChangeListener.java

```diff
@@ -0,0 +1,65 @@
+package com.thinkbiganalytics.search.indexing;
+
+import com.thinkbiganalytics.feedmgr.rest.model.FeedCategory;
+import com.thinkbiganalytics.feedmgr.rest.model.FeedMetadata;
+import com.thinkbiganalytics.feedmgr.service.category.MetadataChangeListener;
+import com.thinkbiganalytics.search.api.Search;
+import com.thinkbiganalytics.search.api.SearchIndex;
+
+import org.apache.commons.beanutils.PropertyUtils;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import java.util.Map;
+
+import javax.inject.Inject;
+
+/*-
+ * #%L
+ * thinkbig-feed-manager-controller
+ * %%
+ * Copyright (C) 2017 ThinkBig Analytics
+ * %%
+ * Licensed under the Apache License, Version 2.0 (the "License");
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') This block of commented-out lines of code should be removed. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3ACommentedOutCodeLine)